src/calculators/sg_correction.rs

```diff
@@ -1,11 +1,28 @@
+//! Is used to correct the specific gravity
+//! reading according to the difference
+//! between the measurement temperature
+//! and the calibration temperature
+
 pub struct SgCorrection;
 
 impl SgCorrection {
+    /// Corrects a specific gravity value with respect to
+    /// a given calibration temperature, measurement
+    /// temperature
+    /// ```
+    /// // rustybeer sg_correction --sg 5 --ct 23 --mt 22
```

Comment:
Here maybe use the function call as an example instead the CLI commans

Reply:
Good point, I'll go ahead and get rid of it